src/components/dashboard/browse/BeanItemDetails.js

```diff
@@ -1,45 +1,65 @@
-import React, { Component, PropTypes } from 'react';
+import React, {Component, PropTypes} from 'react';
 import {Link} from 'react-router';
 import InfiniteList from '../InfiniteList';
+import BackButton from '../BackButton';
 import BeanItemList from './BeanItemList';
 import BeanItemImage from './BeanItemImage';
-import BackButton from '../BackButton';
 
 class BeanItemDetails extends Component {
     render() {
+        const subClass = 'pointer f2 dim br1 ba bw1 tc ph2 mh4 pv3';
+        const subscribe = this.props.bean.inStockBags > 0 && this.props.bean.isActive ?
+            <Link to={'/dashboard/subscribe/' + this.props.params.id} className="no-underline black">
+                <div className={subClass + ' green'}>Subscribe</div>
+            </Link> :
+                <div>
+                    <div className={subClass + ' black-40 black--40'}>Not Available</div>
```

Comment:
roasters manually update `isActive`, but the `bagsInStock` should decrement correctly when orders are created.